### DISCUSSION TITLE
QUESTION ABOUT ENABLING THE WIDGET TO BROWSE FILES-regd.

### DISCUSSION THREAD

**Discussion by KISHORE274-ux** [opengeos/leafmap#817] [Q&A] [answered]
CAN I MAKE THE WIDGETS AVAILABLE WITH GOOGLE SATELLITE BASEMAP TO BROWSE THE FILES IN MY COMPUTER AND LOAD IT ON THE BASEMAP? IF YES, CAN YOU PROVIDE THE SOURCE CODE FOR ACCOMPLISHING THAT?
![Screenshot 2024-07-05 205922](https://github.com/opengeos/leafmap/assets/174684295/b0c54e44-5cde-4c7b-b095-fcb38292a197)
![Screenshot 2024-07-05 205949](https://github.com/opengeos/leafmap/assets/174684295/fd2088c7-f0f2-4050-b6ad-684ff59b31e8)

**Comment by giswqs** [comment, marked as answer]
See https://github.com/opengeos/leafmap/blob/3339b5f7439835782e57c5d340cf04eca5c464dd/leafmap/toolbar.py#L584
